duckduckgo/iOS
# iOS 13 and LHF tweaks

Task/Issue URL: 

* iOS 13 Support, see sub tasks for details: https://app.asana.com/0/414235014887631/1128551253387167
* Update Bookmark Icon: https://app.asana.com/0/414235014887631/812709898398192
* Privacy Toggle Pixel: https://app.asana.com/0/414235014887631/1122812353660909

Tech Design URL:
CC:

**Description**:

LHF Tweaks mostly around preparing for iOS 13.  There's some things we can't do until we move to Xcode 11 full time so they've been deferred.

**Steps to test this PR**:
1. Give the app a good smoke test from a fresh install (onboarding changes), the changes are mostly cosmetic. 
1. Check pixel for toggle privacy works


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
